.github/workflows/python-package.yml

```
@@ -16,7 +16,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
```

actionlint (1.7.7)

expecting a single ${{...}} expression or matrix, but found plain text node

(syntax-check)